auth-method-oauth2.js

```diff
@@ -1138,11 +1138,18 @@ class AuthMethodOauth2 extends AmfHelperMixin(AuthMethodBase) {
     if (!this._hasType(model, sec.OAuth2Settings)) {
       return;
     }
-    this.authorizationUri = this._getValue(model, sec.authorizationUri) || '';
-    this.accessTokenUri = this._getValue(model, sec.accessTokenUri) || '';
-    this.scopes = this._redSecurityScopes(model[this._getAmfKey(sec.scope)]);
+    const flows = this._getValueArray(model, sec.flows) || [];
+    // TODO this is temporary in order to support AMF 4 model change
```

Comment:
What dies this means?
This looks like a breaking change to me
Is the previous version of the model going to be working?